### PR TITLE
chore: remove coinbase

### LINKED ISSUE #2241: Remove "Coinbase pay coming soon" from on/off ramp modal
### Overview

Coinbase has no plans as of now to offer affiliate revs for using their new on-ramp widget. Considering many other providers and potential partners are offering rev share agreements, we have decided not to prioritize the Coinbase on ramp at this time.

### References and additional details

none

### Acceptance Criteria

The "Coming soon" Coinbase pay callout should no longer appear in the fiat on/off ramp modal.

### Need By Date

_No response_

### Screenshots/Mockups

Screen Shot 2022-07-26 at 12 29 56 PM

### Estimated effort

1 hour

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
## Description

this is not proceeding and unfortunately needs to be removed.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #2241 

## Risk

check fox page still functions as intended

## Testing

* coinbase should not appear in fiat ramps list
* feature flag should be gone

## Screenshots (if applicable)

before

image

after

image

### CODE REVIEW

File: src/plugins/foxPage/components/AssetActions.tsx [hunk 1]
```diff
@@ -11,30 +11,23 @@ import {
...
-import { adapters, AssetId, fromAccountId } from '@shapeshiftoss/caip'
+import { AssetId } from '@shapeshiftoss/caip'
 import { foxyAddresses } from '@shapeshiftoss/investor-foxy'
-import { getConfig } from 'config'
 import { FoxyPath } from 'features/defi/providers/foxy/components/FoxyManager/FoxyCommon'
 import qs from 'qs'
-import { useEffect, useState } from 'react'
 import { useTranslate } from 'react-polyglot'
 import { useHistory, useLocation } from 'react-router'
-import coinbaseLogo from 'assets/coinbase-pay/cb-pay-icon.png'
```
Comment (gomesalexandre) [inline src/plugins/foxPage/components/AssetActions.tsx:23]: We can also remove the `src/assets/coinbase-pay` folder from the repo

File: src/plugins/foxPage/components/AssetActions.tsx [hunk 2]
```diff
@@ -106,20 +85,6 @@ export const AssetActions: React.FC<FoxTabProps> = ({ assetId }) => {
...
-                />
-              </Box>
-              <SkeletonText isLoaded={Boolean(description?.length)} noOfLines={3}>
-                <CText color='gray.500' mb={6}>
-                  {coinbasePayFeatureFlag && isFoxAsset
-                    ? translate('plugins.foxPage.purchaseFox')
-                    : trimmedDescription}
-                </CText>
-              </SkeletonText>
-
```
Comment (gomesalexandre) [inline src/plugins/foxPage/components/AssetActions.tsx:122]: We should keep this node, just remove the coinbase flag left side, see prod vs. this diff

image
image

**Review by gomesalexandre — CHANGES_REQUESTED** [review]
Looks good, grep and fzf both tell everything coinbase is removed, except from the coinbase pay logo asset that can be removed

Also a little regression in the Fox Page, there's one smallish hunk that needs reverted with the Coinbase flag logic removed

**Review by 0xApotheosis — COMMENTED** [review]
As @gomesalexandre said, just the `coinbase-pay` repo to go.

But @gomesalexandre the `coinbase.png` logo needs to stay, because we still want the coinbase logo to show in the centralised trading opportunities section I imagine.

File: src/plugins/foxPage/components/AssetActions.tsx (hunk 2 above)
Comment (0xdef1cafe) [reply src/plugins/foxPage/components/AssetActions.tsx:122]: yeah i got a little heavy handed rambo here i'll bring this back

File: src/plugins/foxPage/components/AssetActions.tsx (hunk 1 above)
Comment (0xdef1cafe) [reply src/plugins/foxPage/components/AssetActions.tsx:23]: actually this one needs to stay

**Comment by gomesalexandre** [comment]
> As @gomesalexandre said, just the `coinbase-pay` repo to go.
> 
> But @gomesalexandre the `coinbase.png` logo needs to stay, because we still want the coinbase logo to show in the centralised trading opportunities section I imagine.

That's correct, the `coinbase.png` asset should stay, it is scoped in the foxPage domain and is used as the icon of the centralized opportunity as you've mentioned, linking to Coinbase to get FOX. Only the Coinbase *Pay* one should go.

File: src/plugins/foxPage/components/AssetActions.tsx (hunk 1 above)
Comment (0xdef1cafe) [reply src/plugins/foxPage/components/AssetActions.tsx:23]: oh you're right, killed

**Review by gomesalexandre — APPROVED** [review]
LGTM, tested again and the Fox Page description is back, no regression on the Fox Page coinbase asset icon and everything coinbase pay is removed

🍬

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**